config/esp32/components/chip/Kconfig

```diff
@@ -321,7 +321,7 @@ menu "CHIP Device Layer"
         config CHIP_TASK_STACK_SIZE
             int "CHIP Task Stack Size"
             range 0 65535
-            default 5120
+            default 6144
```

Comment:
Why did this grow?

Reply:
There are stackoverflow issue on CHIP task when sending commands because the stack is a bit deeper then before.